src/components/calcite-date-picker-month-header/calcite-date-picker-month-header.scss

```diff
@@ -1,137 +1,139 @@
 .header {
-  display: flex;
-  justify-content: space-between;
-  padding: 0 3px;
+  @apply flex justify-between;
+  @apply py-0 px-1;
 }
 
 :host([scale="s"]) {
   .text {
-    font-size: 14px;
+    @apply text--1h;
   }
   .chevron {
-    height: 38px;
+    @apply h-10;
   }
 }
 
 :host([scale="m"]) {
   .text {
-    font-size: 16px;
+    @apply text-0h;
   }
   .chevron {
-    height: 48px;
+    @apply h-12;
   }
 }
 
 :host([scale="l"]) {
   .text {
-    font-size: 18px;
+    @apply text-1h;
   }
   .chevron {
-    height: 64px;
+    @apply h-16;
   }
 }
 
 .chevron {
-  color: var(--calcite-ui-text-2);
-  flex-grow: 0;
+  @apply text-color-2
+    flex-grow-0
+    box-border
+    outline-none
+    py-0
+    px-1
+    my-0
+    -mx-1
```

Comment:
py-0 my-0 artifacts from the shorthand? remove?

Reply:
removed